test/test.py

```diff
@@ -512,83 +497,23 @@ def _assert_bundle(self, uuid, version, manifest, metadata_files,
 
         return bundle
 
-    dss_subscription_query = {
-        "query": {
-            "bool": {
-                "must_not": [
-                    {
-                        "term": {
-                            "admin_deleted": True
-                        }
-                    }
-                ],
-                "must": [
-                    {
-                        "exists": {
-                            "field": "files.project_json"
-                        }
-                    },
-                    {
-                        "prefix": {
-                            "uuid": "abc"
-                        }
-                    }, {
-                        "range": {
-                            "manifest.version": {
-                                "gte": "2018-11-27"
-                            }
-                        }
-                    }
-                ]
-            }
-        }
-    }
-
-    # FIXME: Remove test or add replacement
-    @skip('DSS is EOL (https://github.com/DataBiosphere/hca-metadata-api/issues/37)')
-    def test_many_bundles(self):
-        num_workers = os.cpu_count() * 16
-        client = dss_client(num_workers=num_workers)
-        # noinspection PyUnresolvedReferences
-        response = client.post_search.iterate(es_query=self.dss_subscription_query, replica="aws")
-        fqids = [r['bundle_fqid'] for r in response]
-
-        def to_json(fqid):
-            uuid, _, version = fqid.partition('.')
-            version, manifest, metadata_files = download_bundle_metadata(client, 'aws', uuid, version, num_workers=0)
-            bundle = Bundle(uuid, version, manifest, metadata_files)
-            return as_json(bundle)
-
-        with ThreadPoolExecutor(max_workers=num_workers) as tpe:
-            futures = {tpe.submit(to_json, fqid): fqid for fqid in fqids}
-            done, not_done = wait(futures.keys())
-
-        self.assertFalse(not_done)
-        errors, bundles = {}, {}
-        for future in done:
-            exception = future.exception()
-            if exception:
-                errors[futures[future]] = exception
-            else:
-                bundles[futures[future]] = future.result()
-
-        # FIXME: Assert JSON output?
-
-        self.assertEqual({}, errors)
-
-    # FIXME: Remove test or add replacement
-    @skip('DSS is EOL (https://github.com/DataBiosphere/hca-metadata-api/issues/37)')
-    def test_large_bundle(self):
-        _, manifest, _ = download_bundle_metadata(client=dss_client('prod'),
-                                                  replica='aws',
-                                                  uuid='82164816-64d4-4975-a248-b66c4fdad6f8',
-                                                  version='2019-09-26T054644.254919Z')
-        self.assertEqual(755, len(manifest))
+    def test_canned_staging_area(self):
+        repo_owner = 'HumanCellAtlas'
```

Comment:
Would be great if we could parameterize this with a link like

`https://github.com/HumanCellAtlas/schema-test-data/tree/491ab2f51a12e50c9cda6a105b73ac5f308ad684/tests`

or

`https://github.com/HumanCellAtlas/schema-test-data/tree/release-data-24/03/2021/tests`

and parse the `StagingArea` constructor arguments out of the URL. You'd need a heuristic to distinguish between the two forms, say, by checking if the path element after `tree` is a commit hash. For now we only need the first form, the one that uses a commit, so we can post-pone support for the second form.

Either way, this test must refer to a specific commit.